example/gardener-local/gardenlet/values.yaml

```diff
@@ -0,0 +1,85 @@
+global:
+  gardenlet:
+    additionalVolumes:
+    - name: kube-root-ca
+      configMap:
+        name: kube-root-ca.crt
+    additionalVolumeMounts:
+    - name: kube-root-ca
+      mountPath: /var/kube-root-ca
+
+    config:
+      gardenClientConnection:
+        kubeconfigSecret:
+          name: gardenlet-kubeconfig
+          namespace: garden
+        bootstrapKubeconfig:
+          name: gardenlet-kubeconfig-bootstrap
+          namespace: garden
+          kubeconfig: |
+            apiVersion: v1
+            kind: Config
+            current-context: gardenlet-bootstrap
+            clusters:
+            - cluster:
+                certificate-authority: /var/kube-root-ca/ca.crt
+                server: https://kubernetes.default.svc:443
+              name: default
+            contexts:
+            - context:
+                cluster: default
+                user: gardenlet-bootstrap
+              name: gardenlet-bootstrap
+            users:
+            - name: gardenlet-bootstrap
+              user:
+                token: 07401b.f395accd246ae52d
+      controllers:
+        shoot:
+          reconcileInMaintenanceOnly: true
+      featureGates:
+        Logging: true
+        HVPA: true
+        HVPAForShootedSeed: true
+        ManagedIstio: true
+        APIServerSNI: true
+        CachedRuntimeClients: true
+        SeedKubeScheduler: false
+        ReversedVPN: true
+        UseDNSRecords: true
+        DenyInvalidExtensionResources: true
+      seedConfig:
+        apiVersion: core.gardener.cloud/v1beta1
+        kind: Seed
+        metadata:
+          name: local
+        spec:
+          provider:
+            type: local
+            region: local
+          dns:
+            ingressDomain: local.seed.local.gardener.cloud
+          secretRef:
+            name: seed-local
+            namespace: garden
+          networks:
+            nodes: 172.19.0.0/16
+            pods: 10.1.0.0/16
+            services: 10.2.0.0/16
```

Comment:
> I do not think the node CIDR matters because nodes are "pods" with Pod IPs, right?

Yes, that's why the `Shoot` example in `example/provider-local/shoot.yaml` does not specify `.spec.networking.nodes`. However, for the `Seed` we should probably specify it to prevent that conflicting shoot CIDRs can be specified.